src/main/connector/system/Linux/Linux.yaml

```diff
@@ -210,24 +199,20 @@ monitors:
       type: multiInstance
       sources:
         fileSystemInfo:
-          # filesystem;mountpoint;type;used;available
           type: commandLine
-          commandLine: /usr/bin/df -B1 --output=source,target,fstype,used,avail,size
+          commandLine:  df --block-size=1
           computes:
-          - type: awk
-            script: NR > 1 {print $1 "(" $2 ")" ";" $2 ";" $3 ";" $4 ";" $5 ";" $4 / $6 ";" $5 / $6}
+          - type: awk 
+            script: /\/dev\/[shv]d[a-z]|\/dev\/mapper/ {gsub("/dev/", "", $1); gsub("/mapper/", "", $1); print $1 ";" $2 ";" $3 ";" $4 ";" $3 / $2}
       mapping:
         source: ${source::fileSystemInfo}
         attributes:
           id: $1
-          system.filesystem.device: $1
-          system.filesystem.mountpoint: $2
-          system.filesystem.type: $3
         metrics:
-          system.filesystem.usage{system.filesystem.state=used}: $4
-          system.filesystem.usage{system.filesystem.state=free}: $5
-          system.filesystem.utilization{system.filesystem.state=used}: $6
-          system.filesystem.utilization{system.filesystem.state=free}: $7
+          system.filesystem.size.bytes: $2

```

Comment:
This doesn't follow OpenTelemetry semantic conventions. Please make sure to follow [these conventions](https://opentelemetry.io/docs/specs/semconv/system/system-metrics/).